stock_quant_partner_info/models/stock_quant.py

```diff
@@ -14,8 +14,12 @@ class StockQuant(models.Model):
 
     @api.depends('location_id', 'history_ids')
     def _compute_partner_id(self):
-        for quant in self.filtered(
-                lambda q: q.location_id.usage in ['customer', 'supplier']):
-            move = quant.history_ids[:-1]
-            quant.dest_partner_id =\
-                move.picking_id.partner_id.commercial_partner_id
+        for quant in self:
+            moves = quant.history_ids.sorted(key=lambda m: m.date)
+            dest_partner = False
+            if moves:
+                move = moves[-1]
```

Comment:
What's the aim of this ?

Reply:
The original code would not update the dest_partner of a quant which has been returned from customer to stock for RMA for instance as it only operated on quants in customers or suppliers.